knitr/planetary_motion/planetary_motion.rmd

```diff
@@ -0,0 +1,735 @@
+The techniques we deploy involve a combination of mathematical considerations that are specific to the system at hand and statistical principles we can generalize.
+This case study examines the model in the controlled setting of simulated data.
+
+We begin by constructing our model, using elementary notions of classical mechanics.
+After failing to fit this complete model, we develop a simpler model and build our way back to the original model.
+This step-by-step expansion allows us to isolate issues that make Bayesian inference challenging.
+Classic tools we use involve: running multiple chains, convergence diagnostics, and simulating data from the fitted model.
+The latter can be done in several ways and an agile use of plots proves extremely helpful.
+Running simulations for multiple parameter values allows us to understand how the parameters interact with the likelihood, and identify local modes in the posterior.
+In our presentation, we try to distinguish generalizable methods, problem-specific steps, and shortcoming in existing defaults.
+
+
+## R setup {-#setup}
+
+```{r message = FALSE}
+# Adjust to your setting
+.libPaths("~/Rlib/")
+
+library(cmdstanr)
+set_cmdstan_path("~/Rlib/cmdstan/")
+library(rstan)
+library(ggplot2)
+
+library(plyr)
+library(tidyr)
+library(dplyr)
+library(boot)
+library(latex2exp)
+source("tools.r")
+
+set.seed(1954)
+```
+
+All the requisite code to run this notebook can be found online, in the [planetary motion github repository](https://github.com/stan-dev/example-models/tree/case-study/planet/knitr/planetary_motion).
+
+# Building the model
+
+Consider a simple star-planet system.
+We assume the star is much more massive than the planet and approximate the position of the star as _fixed_.
+We would like to estimate the following quantities:
+
+* The gravitational force between the two objects.
+We assume the gravitational constant is known, $G = 1.0 \times 10^{-3}$ in some unit, and aim to evaluate the star-planet mass ratio.
+To do this, we set the planetary mass to $m = 1$.
+It remains to evaluate the solar mass, $M$.
+* The initial position vector, $q_0$, and the initial momentum vector, $p_0$, of the planet.
+* The subsequent position vector, $q(t)$, of the planet over time.
+* The position vector of the star, $q_*$.
+
+## Equations of motion
+
+A more natural treatment of the problem would be using polar coordinates, but for simplicity, let's use Cartesian coordinates.
+The motion of the planet is described by Newton's law of motion, which is a second-order differential equation.
+For convenience, we use Hamilton's formulation of the law, which is based on a system of two first-order differential equations,
+\begin{eqnarray*}
+  \frac{\mathrm d q}{\mathrm d t} & = & \frac{p}{m}, \\
+  \frac{\mathrm d q}{\mathrm d t} & = & - \frac{k}{r^3} (q - q_*),
+\end{eqnarray*}
+where $k = GmM = 10^{-3}M$, and $r = \sqrt{(q - q_*)^T(q - q_*)}$ is the distance between the planet and the star.
```

Comment:
Yeah, I had a better notation, but it didn't render on the figure, so I had to simplify a bit. Definitely capital G, since this is the gravitational constant. m and M are defined earlier in the text.